### ISSUE TITLE
Problems with U+10A39 KHAROSHTHI SIGN CAUDA

### ISSUE THREAD

**Issue by dscorbett** [googlefonts/noto-fonts#1208]
### Font
NotoSansKharoshthi-Regular.ttf

### Where the font came from, and when
Site: https://noto-website-2.storage.googleapis.com/pkgs/NotoSansKharoshthi-unhinted.zip
Date: 2018-04-09

### Font version
Version 1.03 uh

### Issue
U+10A39 KHAROSHTHI SIGN CAUDA should be drawn as a continuation of its base’s stroke, but in Noto Sans Kharoshthi it is clearly a distinct overlaid glyph. When it occurs between a consonant and a vowel, it blocks any consonant–vowel ligature. It overlaps the preceding letter. It is not attached to the vowel sign in the sequence 𐨨𐨹𐨁 _ḿi_. It is slanted upwards, but the examples in [Glass 2000](https://web.archive.org/web/20170829081425/http://www.andrewglass.org/downloads/Glass_2000.pdf) are all more or less horizontal.

### Character data
𐨒𐨹𐨗𐨹𐨜𐨹𐨟𐨹𐨡𐨹𐨤𐨹𐨨𐨹𐨁𐨩𐨹𐨁𐨬𐨹𐨭𐨹𐨯𐨹
U+10A12 KHAROSHTHI LETTER GA
U+10A39 KHAROSHTHI SIGN CAUDA
U+10A17 KHAROSHTHI LETTER JA
U+10A39 KHAROSHTHI SIGN CAUDA
U+10A1C KHAROSHTHI LETTER DDA
U+10A39 KHAROSHTHI SIGN CAUDA
U+10A1F KHAROSHTHI LETTER TA
U+10A39 KHAROSHTHI SIGN CAUDA
U+10A21 KHAROSHTHI LETTER DA
U+10A39 KHAROSHTHI SIGN CAUDA
U+10A24 KHAROSHTHI LETTER PA
U+10A39 KHAROSHTHI SIGN CAUDA
U+10A28 KHAROSHTHI LETTER MA
U+10A39 KHAROSHTHI SIGN CAUDA
U+10A01 KHAROSHTHI VOWEL SIGN I
U+10A29 KHAROSHTHI LETTER YA
U+10A39 KHAROSHTHI SIGN CAUDA
U+10A01 KHAROSHTHI VOWEL SIGN I
U+10A2C KHAROSHTHI LETTER VA
U+10A39 KHAROSHTHI SIGN CAUDA
U+10A2D KHAROSHTHI LETTER SHA
U+10A39 KHAROSHTHI SIGN CAUDA
U+10A2F KHAROSHTHI LETTER SA
U+10A39 KHAROSHTHI SIGN CAUDA

### Screenshot
![𐨒𐨹𐨗𐨹𐨜𐨹𐨟𐨹𐨡𐨹𐨤𐨹𐨨𐨹𐨁𐨩𐨹𐨁𐨬𐨹𐨭𐨹𐨯𐨹](https://user-images.githubusercontent.com/1124347/40888706-fc73bb3c-6728-11e8-8361-35c25471fdc9.png)

**Comment by simoncozens** [comment]
This appears to have been fixed.

**Comment by dscorbett** [comment]
The caudas still mostly don’t look like they do in Andrew Glass’s paper. Is the Noto design attested and intentional?

**Comment by simoncozens** [comment]
Good point. I wasn't commenting on the design so much as the engineering; they no longer overlap with the following glyph, no longer block conjuncts, and don't attach to ḿi. The actual outline shape looks... upside down compared to the palaeography?

<img width="79" alt="Screenshot 2022-02-18 at 16 07 42" src="https://user-images.githubusercontent.com/106728/154718930-e60529c5-1f70-4aaf-9f42-243ad40df6b1.png"><img width="159" alt="Screenshot 2022-02-18 at 16 08 11" src="https://user-images.githubusercontent.com/106728/154719023-842ed7d1-29a6-4822-8866-535469ba7fb3.png">
